src/gui/curses/normal/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ set(LIB_GUI_CURSES_SRC
 
 find_package(Ncurses)
 if(NCURSES_FOUND)
+  if(NCURSES_CFLAGS)
```

Comment:
Is this condition really needed?
Calling `add_definitions()` should be allowed and harmless in CMake.

Reply:
I was worried that `add_definitions` might be unhappy with an empty string, I'll check this.

Reply:
Removed the condition.